stderr of test 'python_dbapi_embedded` in directory 'src/test/mapi` itself:


# 13:29:01 >  
# 13:29:01 >  Mtimeout -timeout 180 ./python_dbapi_embedded.sh python_dbapi_embedded 
# 13:29:01 >  


# 13:29:01 >  
# 13:29:01 >  sqlsample.py /ufs/niels/scratch/rc/Linux-x86_64/var/MonetDB4/dbfarm mTests_src_test_mapi
# 13:29:01 >  


# 13:29:02 >  
# 13:29:02 >  Done.
# 13:29:02 >  

